### DISCUSSION TITLE
Personal access tokens

### DISCUSSION THREAD

**Discussion by cynex-k** [community/community#34821] [Apps, API and Webhooks] [answered]
hey.
I follow the steps for generate a token from github docs but when I click the generate button it doesn't shows me the generated token ?

**Comment by jfullstackdev** [comment]
just tested it by now, there is no problem creating it

![image](https://user-images.githubusercontent.com/47092464/193440707-9a6f20f2-214a-41c9-9b6c-0df07100e538.png)

**Comment by DhananjayPorwal** [comment, marked as answer]
You can access the token key once you created. After that, you can't see the token key. One thing you can do is, regenerate the token key and store it. You can regenerate `n` number of times token key
![image](https://user-images.githubusercontent.com/51775507/193445140-4d62cd47-a7a8-445a-9a7c-d805f2f62cac.png)

> PS: If it helps, accept this comment as an answer

**Reply by JESY9292** [reply]
Www.Jesy.com

**Reply by JESY9292** [reply]
Www.Jesy.com

**Comment by JESY9292** [comment]
https://Jesy.com

**Comment by Shinam** [comment]
Hi,
Each time I generate or regenerate a token, I never got the key as jdevstatic can show, so I'm unable to copy it, any way to fix it ? Thanks

<img width="820" alt="Capture d’écran 2022-10-28 à 02 42 05" src="https://user-images.githubusercontent.com/9668950/198423173-fec9fb68-235a-4933-9fba-1f295a5adbfa.png">

**Reply by Vikingama** [reply]
disabled the adblock, plz
